lib/components/user/stacked-pane-display.tsx

```diff
@@ -10,8 +9,15 @@ import FormNavigationButtons from './form-navigation-buttons'
  *
  * TODO: add types once Pane type exists
  */
-// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
-const StackedPaneDisplay = ({ onCancel, paneSequence, title }) => {
+const StackedPaneDisplay = ({
+  onCancel,
+  paneSequence,
+  title
+}: {
+  onCancel: () => void
```

Comment:
In the tech call a few weeks ago, I think we decided to consistently include explicit types for the props of React components. This is what we meant by that right (defining after props in a list, rather than exporting to a type like `StackedPaneDisplayProps`?) Not raising an issue here just using this as a use case to make sure I understand.

Reply:
Yep! 2f4185241c0e6bcd1c7a030edf1b8a2eb074f713 is how this would look following our new agreed-upon pattern

Reply:
Gucci. Thanks for the explanation